xarray/core/common.py

```diff
@@ -1076,7 +1076,8 @@ def where(self, cond: Any, other: Any = dtypes.NA, drop: bool = False) -> Self:
             If a callable, it must expect this object as its only parameter.
         other : scalar, DataArray or Dataset, optional
             Value to use for locations in this object where ``cond`` is False.
-            By default, these locations filled with NA.
```

Comment:
Add callable to the accepted types in the docsting

Reply:
thanks, done